src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,48 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
+  return (
+    <Card variant='outline' my={1}>
+      <Stack
+        direction={{ base: 'column', xl: 'row' }}
+        alignItems='center'
+        justifyContent='space-between'
+        px={10}
+        py={4}
+      >
+        <Flex alignItems='center'>
+          <RawText alignSelf='end' fontSize={30} mr={-2} mb={{ base: '2', xl: '-2' }} zIndex={1}>
+            🚜
+          </RawText>
+          <AssetIcon mr={-2} boxSize='40px' src={ethLogoSrc} />
+          <AssetIcon boxSize='40px' src={foxLogoSrc} />
+          <Text ml={5} fontWeight='medium' fontSize='lg' translation='defi.earnCopy' />
```

Comment:
```suggestion
          <Text
            ml={5}
            fontWeight='normal'
            fontSize={{ base: 'md', md: 'lg' }}
            translation='defi.earnCopy'
          />
```

In order to respect mockups